API/utils/util.go

```diff
@@ -35,30 +38,40 @@ const (
 	CORRIDOR
 	PWRPNL
 	GROUP
-	// non hierarchal templates
+	// logical non hierarchical entities
 	ROOMTMPL
 	OBJTMPL
 	BLDGTMPL
+	TAG
 )
 
 type Namespace string
 
 const (
 	Any            Namespace = ""
 	Physical       Namespace = "physical"
+	PStray         Namespace = "physical.stray"
+	PHierarchy     Namespace = "physical.hierarchy"
 	Organisational Namespace = "organisational"
 	Logical        Namespace = "logical"
+	LObjTemplate   Namespace = "logical.objtemplate"
+	LBldgTemplate  Namespace = "logical.bldgtemplate"
+	LRoomTemplate  Namespace = "logical.roomtemplate"
+	LTags          Namespace = "logical.tag"
 )
 
 const HN_DELIMETER = "."           // hierarchyName path delimiter
 const NAME_REGEX = "\\w(\\w|\\-)*" // accepted regex for names that compose ids
 const RESET_TAG = "RESET"          // used as email to identify a reset token
+const HIERARCHYOBJS_ENT = "hierarchy_object"
 
 type RequestFilters struct {
-	FieldsToShow []string `schema:"fieldOnly"`
-	StartDate    string   `schema:"startDate"`
-	EndDate      string   `schema:"endDate"`
-	Limit        string   `schema:"limit"`
+	FieldsToShow []string  `schema:"fieldOnly"`
```

Comment:
When use fieldOnly for /all endpoints, tags are also returned.

Reply:
Fixed in e0add398